_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,444 @@
+
+### Contribute to the knowledge society
+
+Sharing the GC's source code as information that others can learn from or reuse is our civic responsibility.
+It lowers costs for society.
+It increases knowledge.
+
+The Directive on Open Government reflect the value of documents created by public organisations and encourage their reuse.
+Like data and information, source code constitutes a vast, diverse and valuable pool of resources that can benefit the knowledge society.
+Sharing will help maximise the growth potential of the digital economy, as communicated in the Digital Nation Charter's open market:
+
+> Promote competition in digital markets, for all enterprises regardless of size, including in government procurement, encourage and support a dynamic start-up/scale-up culture in the digital and technology sectors, and promote sustainable economic growth through open markets.
+
+This is why the strategy aims to make it possible for ESDC to share software using a process comparable to that for its documents.
+
+Note:
+
+Work done in the open supports greater transparency and accountability, ensuring that citizens can trust the public service in doing due diligence.
+
+It helps contribute to the greater good by sharing knowledge and to ensure perenniality of investments done with our citizens money.
+
+### Build a world class public service
+
+Open standards and OSS are about co-creation and building public services that connect seamlessly across organisational silos and borders.
+That is why it is a component in most of the actions in the ISA2 programme.
+Building a responsible open Canada is something we can only do ourselves and the best way is to do it together.
+
+A [Study on open source software governance at the European Commission](https://op.europa.eu/en/publication-detail/-/publication/1aa67278-5464-11ea-aece-01aa75ed71a1) shows that their open source strategy is in line with the best practices from public services and private companies.
+It confirms that the strengthened policy on the use of open source software is the norm nearly everywhere.
+The study's high-level recommendations are:
+
+- emphasise the use and benefits of open source;
+- create a dedicated open source entity that fosters and measures strategy adoption;
+- improve procurement and product-management processes;
+- establish an open culture;
+- collaborate with communities and the open source software ecosystem;
+- manage legal/licence/intellectual property rights issues; and
+- enhance and develop the technical infrastructure and services.
+
+Note:
+
+The ESDC Open First Strategy builds on the GC Policy and Digital Standards and is in line with the EC open source strategy.
+
+The rest of the world has realized the value of working in the open by default.
+Canada has a policy in line with the world's best approaches.
+
+By adopting this strategy, ESDC is taking a stand and moves towards a world class public service.
+
+## 5. Governing Principles
```

Comment:
Add reference to OSS Guidelines (TBS) and OSS Framework (ESDC) to support the principles.